### PR TITLE
Bursary content added to visa page

### PR DESCRIPTION
**Pull request by laurahermionetennant92**
Bursary content added to visa page

### CODE REVIEW

File: app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md [hunk 1]
```diff
@@ -76,6 +76,12 @@ You’ll be able to apply for your visa if you:
 * [have enough money to support yourself and pay for your course](https://www.gov.uk/student-visa/money)
 * [meet certain standards in speaking, reading, writing and understanding English](https://www.gov.uk/student-visa/knowledge-of-english)
 
+### How a teacher training bursary or scholarship can help with your student visa application
+
+If you are training to teach physics or languages, you may be eligible for a [bursary or scholarship](/non-uk-teachers/fees-and-funding-for-non-uk-trainees/#financial-support-for-non-uk-trainee-languages-and-physics-teachers). Bursaries and scholarships are tax-free payments from the UK government which you do not have to pay back. You can use them in your student visa application as proof that you have enough money to support yourself.
```
Comment (philspray) [inline app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md:81]:
```suggestion
 [bursary or scholarship](/non-uk-teachers/fees-and-funding-for-non-uk-trainees/#financial-support-for-non-UK-trainee-languages-and-physics-teachers).
```

File: app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md [hunk 2]
```diff
@@ -76,6 +76,12 @@ You’ll be able to apply for your visa if you:
 * [have enough money to support yourself and pay for your course](https://www.gov.uk/student-visa/money)
 * [meet certain standards in speaking, reading, writing and understanding English](https://www.gov.uk/student-visa/knowledge-of-english)
 
+### How a teacher training bursary or scholarship can help with your student visa application
```
Comment (philspray) [inline app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md:79]:
```suggestion
### How a bursary or scholarship can help with your student visa application
```

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md [hunk 3]
```diff
@@ -7,9 +7,11 @@ related_content:
     Salaries and benefits of teaching in England : "/salaries-and-benefits"
     A day in the life of a teacher in England : "/a-day-in-the-life-of-a-teacher"
 promo_content:
-navigation: 20.30
-navigation_title: Fees and funding for non-UK trainees
-navigation_description: Learn more about the costs of teacher training in England and eligibility for financial help.
+
+navigation: 20.10
+navigation_title: Fees and financial support for non-UK trainee teachers
+navigation_description: Learn more about teacher training fees and help from the UK government for physics and languages trainees.
```
Comment (philspray) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md:13]:
```suggestion
navigation_description: Learn more about teacher training fees in England and financial help for physics and languages trainees.
```

**Review by philspray — COMMENTED** [review]
Just a few suggestions and a fix for the link @laurahermionetennant92

File: app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md (hunk 1 above)
Comment (laurahermionetennant92) [reply app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md:81]: I think lower case is correct as the link is now working?

**Review by philspray — APPROVED** [review]
Looks ok to me

**Pull request merged**